index.html

```diff
@@ -201,15 +202,34 @@ <h3 class="CSlist-title">
       </ul>
       <h3 class="CSlist-title">
         Frameworks
-        <i class="arrowhead right-direction arrow-margin"></i>
+        <i class="arrowhead right-direction"></i>
       </h3>
       <hr class="horizontal-line">
       <h3 class="CSlist-title">
         Skills
-        <i class="arrowhead right-direction arrow-margin"></i>
+        <i class="arrowhead right-direction"></i>
       </h3>
       <hr class="horizontal-line">
     </div>
   </section>
+  <!-- Footer -->
+  <section class="container-bg-color">
+    <footer class="footer-bg">
+      <h2 class="pg-headings white-txtcolor poppins-font" id="footer-hreading">
+        Contact me
+      </h2>
+      <p class="poppins-font footer-p">
+        If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+      </p>
+      <form action="https://formspree.io/f/xwkydobk" id="form" method="post">
+        <input class="form-field" type="text" id="username" name="username" maxlength="30"
+        placeholder="Full Name" required>
+        <input class="form-field" type="email" id="user-email" name="user-email" placeholder="Email Address" required>
+        <textarea class="form-field" name="message" maxlength="500" rows="7" cols="70" id="form-msg" placeholder="Enter text here." required></textarea>
+        <button type="button" class="poppins-font button" id="footerbtn">Get It Touch</button>
+      </form>
```

Comment:
- [ ] `required`
- [ ] Amazing job you have implementing the form so far.
- [ ] please make sure all the validation are working
- [ ] also making sure the form is posting after pressing the submit button. please see screenshots below

`if your posting is working fine you will be able to get this message`
![Screenshot from 2022-07-07 12-58-36](https://user-images.githubusercontent.com/66230306/177890534-7439fab2-5d8c-40d9-a7d2-345177e3d40f.png)

`amd if your validation is working you should be able to get such messages`
![Screenshot from 2022-07-07 13-00-42](https://user-images.githubusercontent.com/66230306/177890759-060fa495-91c5-4af1-bdce-d34169aaea9c.png)

please see the [form spree](https://formspree.io/html/) for further configuration

Reply:
@mwanawabangona Thanks for reviewing my code. I have fixed the issue now